idseq_dag/steps/run_star.py

```diff
@@ -1,12 +1,218 @@
+import multiprocessing
+import os
+
 from idseq_dag.engine.pipeline_step import PipelineStep
 import idseq_dag.util.command as command
+import idseq_dag.util.s3 as s3
+import idseq_dag.util.log as log
 
 
 class PipelineStepRunStar(PipelineStep):
+    max_input_reads = 75 * 1000 * 1000
```

Comment:
Constants should be uppercase. 
Maybe for later: shouldnt this be move to a centralize parameter management (e.g. AWS parameter store and maybe environment variables)?